docs/navigation-parameters-reference.md

```diff
@@ -1,294 +1,305 @@
-# Navigation parameters reference
-You can use the listed parameters and functions to configure your navigation structure. The examples show how to use selected options.
+# Navigation properties reference
 
-## Navigation configuration example
+> NOTE: To see the navigation properties in use, check out the [navigation configuration example](navigation-configuration-example.md) or configure a test application in the [Luigi fiddle](https://fiddle.luigi-project.io).
 
-This code sample demonstrates a sample structure with the parameters you can use when configuring navigation for Luigi.
+You can use properties and functions in this reference to configure your Luigi navigation structure.
 
+* [Routing properties](#routing-properties)
+* [Navigation properties](#navigation-properties)
+* [Node properties](#node-properties)
+* [Profile](#profile)
+* [Context switcher](#context-switcher)
+* [Product switcher](#product-switcher)
+* [App switcher](#app-switcher)
+
+
+## Routing properties
+You can configure the way Luigi tackles routing in your application in the `routing:` section of the configuration file. For example, you can choose the routing strategy to apply in your application as either hash or path location routing.
+
+### useHashRouting
+- **type**: boolean
+- **description**: defines either hash-based (`url.com/#/yourpath`) or path-based (`url.com/yourpath`) routing.
+
+### nodeParamPrefix
+- **type**: string
+- **description**: sets the prefix character when using the `LuigiClient.linkManager().withParam()` function, which provides a way to simply attach query properties to the view URL for activities such as sorting and filtering. The URL contains the properties to allow deep linking. If you want to use a different character prefix, define yours here. The default character is `~`.
```

Comment:
We should explain that the prefix here is necessary to avoid potential conflicts with query parameters belonging to the Luigi app itself. Only query parameters with that prefix are passed to microfrontends.